Core/Core/DeveloperMenu/ExperimentalFeaturesViewController.swift

```diff
@@ -0,0 +1,62 @@
+//
+// This file is part of Canvas.
+// Copyright (C) 2019-present  Instructure, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+//
+
+import UIKit
+
+public class ExperimentalFeaturesViewController: UITableViewController {
+    public var afterToggle: (() -> Void)?
+
+    override public func viewDidLoad() {
+        super.viewDidLoad()
+
+        let barButtonItem = UIBarButtonItem(title: "Toggle All", style: .plain, target: self, action: #selector(toggleAll(_:)))
+        navigationItem.rightBarButtonItem = barButtonItem
+
+        title = "Experimental Features"
+        tableView?.register(SwitchTableViewCell.self, forCellReuseIdentifier: String(describing: SwitchTableViewCell.self))
+    }
+
+    override public func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return ExperimentalFeature.allFeatures.count
+    }
+
+    override public func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let feature = ExperimentalFeature.allFeatures[indexPath.row]
+        let cell = tableView.dequeue(for: indexPath) as SwitchTableViewCell
+        cell.textLabel?.text = feature.remoteConfigKey
+        cell.toggle.tag = indexPath.row
+        cell.toggle.isOn = feature.isEnabled
+        cell.toggle.addTarget(self, action: #selector(toggleFeature(_:)), for: .valueChanged)
+        return cell
+    }
+
+    @objc
+    func toggleFeature(_ sender: UISwitch) {
+        ExperimentalFeature.allFeatures[sender.tag].isEnabled = sender.isOn
+        if sender.isOn == true {
+            afterToggle?()
```

Comment:
don't we need this to be called even if a feature just got disabled?

Reply:
Ah yeah I removed that from the toggle all but forgot about it here.